scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/descriptor/JobDescriptorImpl.java

```diff
@@ -83,14 +84,14 @@
 
     /** Job tasks to be able to be schedule */
     @XmlTransient
-    private Map<TaskId, EligibleTaskDescriptor> eligibleTasks = new ConcurrentHashMap<>();
+    public Map<TaskId, EligibleTaskDescriptor> eligibleTasks = new ConcurrentHashMap<>();
 
     /** Those are not directly eligible, and will be triggered by an IF control flow action */
     @XmlTransient
     private Map<TaskId, EligibleTaskDescriptor> branchTasks = new ConcurrentHashMap<>();
 
     /** Job running tasks */
-    private Map<TaskId, TaskDescriptor> runningTasks = new ConcurrentHashMap<>();
+    public Map<TaskId, TaskDescriptor> runningTasks = new ConcurrentHashMap<>();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make runningTasks a static final constant or non-public and provide accessors if needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)